Packs/Core/Playbooks/playbook-Credential_Dumping_using_a_known_tool.yml

```diff
@@ -0,0 +1,628 @@
+      condition:
+      - - operator: containsString
+          left:
+            value:
+              complex:
+                root: Core.Endpoint
+                accessor: endpoint_type
+            iscontext: true
+          right:
+            value:
+              simple: WORKSTATION
+          ignorecase: true
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Core.Endpoint
+                accessor: endpoint_status
+            iscontext: true
+          right:
+            value:
+              simple: CONNECTED
+          ignorecase: true
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.Endpoint.is_isolated
+            iscontext: true
+          right:
+            value:
+              simple: AGENT_UNISOLATED
+          ignorecase: true
+    - label: Already isolated
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.Endpoint.is_isolated
+            iscontext: true
+          right:
+            value:
+              simple: AGENT_ISOLATED
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 10,
+          "y": 10
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: e779dfc7-1f39-4ea1-8395-693901916095
+    type: regular
+    task:
+      id: e779dfc7-1f39-4ea1-8395-693901916095
+      version: -1
+      name: Terminate Causality (CGO)
+      description: Terminate a process by it's causality ID.
+      script: '|||core-terminate-causality'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "33"
+      '#none#':
+      - "28"
+    scriptarguments:
+      agent_id:
+        simple: ${alert.agentid}
+      causality_id:
+        simple: ${alert.cid}
+      timeout_in_seconds:
+        simple: "180"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": -650
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: f82b7572-1a45-4d6a-84b5-b6802dcc44af
+    type: regular
+    task:
+      id: f82b7572-1a45-4d6a-84b5-b6802dcc44af
+      version: -1
+      name: Terminate Causality Process Manually
+      description: "Dear Analyst,\n\nDuring the remediation process, the playbook failed to terminate the causality process ${alert.cgoname} \n\nPlease manually terminate the process if possible. \nNote that the next remediation step, if possible, will be endpoint isolation."
```

Comment:
"Note that the next remediation step, if possible, will be endpoint isolation.""
Why do we need it?

Reply:
Yarden asked me to add this line.